### ISSUE TITLE
Vector of consts

### ISSUE THREAD

**Issue by jpbrodsky** [sandialabs/InterSpec#3]
https://github.com/sandialabs/InterSpec/blame/master/InterSpec/D3SpectrumDisplayDiv.h#L389

g++ 4.8.5 will not accept a vector of consts as items in a std::vector must be assignable.

**Comment by jpbrodsky** [comment]
Some other parts of InterSpec require c++14 features not compatible with gcc4.8.5. So, compatibility with 4.8.5 may not be a useful goal. I'm not sure if a vector of consts is allowed in later versions of gcc.

**Comment by wcjohns** [comment]
I had thought this was valid C++11, but anyway it has been fixed.  I have been trying to primarily keep things to C++11 (mainly for MSVS 2012), but I guess I slipped a little after upgrading compilers.

BTW, by default I had the CMake `USE_SPECTRUM_CHART_D3` set to `ON`, however this is still a feature I am slowly working on, so I have changed it to `OFF` by default, and would suggest doing the same for for your local copy (you might have to do this via CMake, I'm not sure if the change to CMakeLists.txt will be picked up in existing builds).  Thanks!
